<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProgresoOpe</class>
 <widget class="QDialog" name="DialogProgresoOpe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelCnp">
       <property name="text">
        <string>Cnp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarCnp">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelCnp">
       <property name="text">
        <string>cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelMet">
       <property name="text">
        <string>Met</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarMet">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelMet">
       <property name="text">
        <string>cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelOrto">
       <property name="text">
        <string>Orto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarOrto">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelOrto">
       <property name="text">
        <string>cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
